h2o-core/src/main/java/water/util/VecUtils.java

```diff
@@ -184,6 +183,53 @@ public static Vec toNumericVec(Vec src) {
     }
   }
 
+  public static Vec toDoubleVec(Vec src) {
+    switch (src.get_type()) {
+      case Vec.T_CAT:
+        return categoricalToDouble(src);
+      case Vec.T_STR:
+        return stringToDouble(src);
+      case Vec.T_NUM:
+      case Vec.T_TIME:
+      case Vec.T_UUID:
+        if (src.isInt())
+          return intToDouble(src);
+        else
+          return src.makeCopy(null, Vec.T_NUM);
```

Comment:
If it's have operation, can we avoid it and return just `src`?

Reply:
it is changed.